src/components/Home/Add Item/AddItemForm/index.js

```diff
@@ -0,0 +1,212 @@
+                                            <label className="modal-subhead" htmlFor="exampleInputEmail1">Title</label>
+                                            <div className="form-row">
+                                                <input type="email" className="form-control col-7 mr-3 ml-1" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter title" required></input>
+                                                <input type="number" className="form-control col" id="inputQuantity" aria-describedby="quantityHelp" placeholder="Quantity" required></input>
+                                            </div>
+                                        </div>
+                                        <div className="form-group">
+                                            <label className="modal-subhead" htmlFor="exampleInputPassword1">Description</label>
+                                            <textarea className="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+                                        </div>
+                                        <button className="btn btn-link addPhotoButton">
+                                            Add Photo
+                                </button>
+                                        <div className="form-group">
+                                            <div className="form-group">
+                                                <label className="modal-subhead" htmlFor="exampleInputEmail1">Deadline Date</label>
+                                                <div className="calendarGroup">
+                                                    {<Calendar minDate={new Date()} onChange={this.handlePaymentCalendarChange} />}
+                                                </div>
+                                            </div>
+                                            <div className="form-group">
+                                                <label className="modal-subhead" htmlFor="exampleInputEmail1">Assign Users</label>
+                                                {<AssignedUsers id={"users" + this.props.type} users={this.props.users} handleUserChange={this.handlePaymentUserChange} />}
+                                            </div>
+                                        </div>
+                                    </form>
+                                    <div className="modal-footer">
+                                        <button type="button" className="btn btn-secondary" data-dismiss="modal">Close</button>
+                                        <button type="button" className="btn btn-primary" data-dismiss="modal">Add Item</button>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                )
+            default:
+                break
```

Comment:
This is written terribly and definitely needs to be rewritten

Reply:
This is where I'll put the code for history when it's done, do you want me to hold off adding it until this has been rewritten?

Reply:
I'd toss history in a different branch for now then we'll do a PR on that when the time comes